caffe2/core/blob_test.cc

```diff
@@ -538,7 +538,13 @@ TEST(TensorTest, Tensor64BitDimension) {
   EXPECT_EQ(tensor.ndim(), 1);
   EXPECT_EQ(tensor.dim(0), large_number);
   EXPECT_EQ(tensor.size(), large_number);
-  EXPECT_TRUE(tensor.mutable_data<char>() != nullptr);
+  try {
+    EXPECT_TRUE(tensor.mutable_data<char>() != nullptr);
```

Comment:
Thanks! You might want to wrap line 552 as well.

Reply:
Thanks! I've updated that.

Reply:
@Yangqing looks to me `Resize` doesn't allocate memory